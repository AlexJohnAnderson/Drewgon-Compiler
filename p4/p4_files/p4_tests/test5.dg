void main(){
	int a;
	a = 15;
	return a;
}	