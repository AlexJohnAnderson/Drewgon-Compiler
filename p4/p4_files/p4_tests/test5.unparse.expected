void main(->void)(){
    int a(int);
    a(int) = 15;
    return a(int);
}